.github/workflows/build-prs-trigger.yaml

```diff
@@ -0,0 +1,28 @@
+name: Trigger build images for PRs
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - closed
```

Comment:
For closed, It will trigger another workflow `Build images for PRs` and delete PR image which was built while PR open.
For reopen, It will trigger the workflow for reopen PR. Sot sure about it, this case will be there or not. I follow the same what we have in `datascience-pipelineoperator`. We can avoid for `reopen` if any concern.